superset-frontend/src/explore/controlPanels/Table.js

```diff
@@ -183,6 +183,21 @@ export default {
             },
           },
         ],
+        [
+          {
+            name: 'hide_cell_bars',
```

Comment:
I'd say make this a default on control called "Display metrics with bars" to [avoid a double negative.](https://learningenglish.voanews.com/a/everyday-grammar-double-negatives/2743416.html).

1. It's more intuitive to make all visibility control about show something optional.
2. We should probably disable this bar chart by default anyway.

Reply:
I wrestled with the same thought, and built it that way initially. I inverted it, thinking that we'd have to do a migration for existing charts so people don't have to go toggle it "on" for every existing chart... but... I wound up setting a default for that prop anyway, so uh... I'll switch it back. Thanks for taking a critical look.